Assets/Mirror/Examples/FPS/Scripts/FPSMove.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+
+namespace Mirror.Examples.FPS
+{
+    public class FPSMove : NetworkBehaviour
+    {
+        public CharacterController controller;
+        public float speed = 300;
+        public float rotationSpeed = 400;
+        public Transform head;
+        float pitch;
+
+        void Start()
+        {
+            // Turn on camera for local player
+            if (hasAuthority) head.gameObject.SetActive(true);
+        }
+
+        void Update()
+        {
+            // movement for local player
+            if (!isLocalPlayer) return;
+
```

Comment:
please try to hide the cursor while playing (or until pressing escape). showing the cursor makes it very difficult to look around and play without clicking on editor buttons.